packages/components/tree-v2/src/composables/useTree.ts

```diff
@@ -213,8 +218,8 @@ export function useTree(props: TreeProps, emit) {
     }
   }
 
-  function handleNodeCheck(node: TreeNode, checked: boolean) {
-    toggleCheckbox(node, checked)
+  function handleNodeCheck(node: TreeNode, checked: CheckboxValueType) {
+    toggleCheckbox(node, checked as boolean)
```

Comment:
Instead of using type assertion here, we can change the type of the second parameter of function `toggleCheckbox` to `CheckboxValueType`.